src/pages/index.js

```diff
@@ -27,45 +31,61 @@ const profileDescriptionInput = document.getElementById(
   "profile-description-input"
 );
 
-const userInfo = new UserInfo(userProfileSelectors);
+const avatarEditButton = document.querySelector(".profile__avatar-edit");
+const avatarEditForm = document.getElementById("avatar-edit-form");
+const avatarUrlInput = document.getElementById("avatar-url-input");
 
-const profilePopup = new PopupWithForm("#profile-edit-modal", {
-  handleFormSubmit: (formData) => {
-    userInfo.setUserInfo({
-      name: formData.name,
-      job: formData.about,
-    });
-    profilePopup.close();
-  },
+const userInfo = new UserInfo({
+  nameSelector: ".profile__title",
+  jobSelector: ".profile__description",
+  avatarSelector: ".profile__image",
 });
-profilePopup.setEventListeners();
 
-const cardPopup = new PopupWithForm("#add-place-modal", {
-  handleFormSubmit: (formData) => {
-    const cardElement = createCard({
-      name: formData.title,
-      link: formData.link,
-    });
+function handleCardClick(name, link) {
+  imagePopup.open({ name, link });
+}
 
-    cardSection.addItem(cardElement);
-    cardPopup.close();
-  },
-});
-cardPopup.setEventListeners();
+function handleDeleteCard(cardId, card) {
+  if (!cardId) {
+    console.error("Card ID is invalid or missing.");
+    return;
+  }
 
-const imagePopup = new PopupWithImage("#image-view-modal");
-imagePopup.setEventListeners();
+  confirmationPopup.open(() => {
+    api
+      .deleteCard(cardId)
+      .then(() => {
+        card._handleDelete();
```

Comment:
`_handleDelete`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.

**Please, correct it everywhere in the code of `index.js`**